scripts/review/diff.ts

```diff
@@ -28,10 +31,67 @@ function parseOptions(): Options {
   };
 }
 
+/**
+ * Check if the changes in the diff patch are of the given types.
+ *
+ * @param changes The changes in the diff patch.
+ * @param types The types of changes to check.
+ */
```

Comment:
Nitpick: the JSDoc documents function parameters but doesn't document the return value. The same applies to other JSDocs in the file